pkg/ctl/scale/nodegroup.go

```diff
@@ -18,20 +18,25 @@ func scaleNodeGroupCmd(g *cmdutils.Grouping) *cobra.Command {
 	ng := cfg.NewNodeGroup()
 
 	cmd := &cobra.Command{
-		Use:   "nodegroup",
+		Use:   "nodegroup NAME",
 		Short: "Scale a nodegroup",
-		Run: func(_ *cobra.Command, args []string) {
+		RunE: func(_ *cobra.Command, args []string) error {
+			name := cmdutils.GetNameArg(args)
+			if name != "" {
+				ng.Name = name
+			}
 			if err := doScaleNodeGroup(p, cfg, ng); err != nil {
 				logger.Critical("%s\n", err.Error())
 				os.Exit(1)
 			}
+			return nil
 		},
 	}
 
 	group := g.New(cmd)
 
 	group.InFlagSet("General", func(fs *pflag.FlagSet) {
-		fs.StringVarP(&cfg.Metadata.Name, "name", "n", "", "EKS cluster name")
+		fs.StringVarP(&cfg.Metadata.Name, "cluster", "n", "", "EKS cluster name")
```

Comment:
We should use `--name` for nodegroup name.

Reply:
Thanks. Addressed in ff91a7d.